### DISCUSSION TITLE
Ex 2 part 1: reuse of Majority Voting

### DISCUSSION THREAD

**Discussion by louxsd** [sebastian-hofstaetter/teaching#150] [Advanced IR: Exercise Q&A]
It is clearly stated that we are not suppose to use Majority Voting to aggregate judgements but a more sophisticated method. Can we use majority voting as a base in our implementation and add other parameters in the aggregation? Or are we supposed to use a completly different method?
Thanks.

**Comment by breezy11** [comment]
You can definitely build on majority voting, but add more parameters to make it more sophisticated. The goal is to make the aggregation process more nuanced and reliable, not just a straightforward vote. As long as you justify your method and show improvements, it should be fine.

**Reply by louxsd** [reply]
Understood, thank you.
